lib/Pipes/TaggedFunctionKind.cpp

```diff
@@ -66,3 +59,18 @@ void TaggedFK::getInvalidations(const Context &Ctx,
 
   // ToRemove.push_back(Target("0x1000", *this));
 }
+
+static void getAllFunctions(const TupleTree<model::Binary> &Model,
+                            pipeline::TargetsList &Out,
+                            const pipeline::Kind &K) {
+
+  for (const auto &Function : Model->Functions) {
+    Out.push_back(Target(Function.Entry.toString(), K));
+  }
+}
+
+void TaggedFunctionKind::allTargets(const pipeline::Context &Ctx,
+                                    pipeline::TargetsList &Out) const {
+  const auto &Model = getModelFromContext(Ctx);
+  getAllFunctions(Model, Out, *this);
```

Comment:
Inline function call

Reply:
not sure what you mean. Inlining this function will do nothing, it is virtual

Reply:
`getAllFunctions` is defined right above but introduces like 10 lines that could be inlined.